### PR TITLE
fixed dead links, other cleanup

### PR DESCRIPTION
**Pull request by tannazie**
Hang found a bunch of dead links and other issues in the process of localizing some docs in the dev docs site. This pull request addresses all of his notes in the following JIra tickets:

https://dts.autodesk.com/browse/SHOT20-99
https://dts.autodesk.com/browse/SHOT20-105
https://dts.autodesk.com/browse/SHOT20-106
https://dts.autodesk.com/browse/SHOT20-107

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/8bba97e31e373d75dfebb21f88acb0d0872354c0/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/442b3a35c0ceeb3a0910db4ae5ebe8efcace82e2/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/794396ed94835657d1eb4cf05986e5d1bde83040/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/dd540041b2527d71b7be8078f497a617ac7a5f3d/index.html)

File: docs/en/toolkit/learning-resources/guides/editing_app_setting.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ This reference box shows details about the configuration settings that control t
 
 Look under the **Configuration** header to find the settings for this specific environment. 
 
-![Configuration_ref](./images/editing_app_setting/10_configuration_ref.png "View task button")
+![Configuration_ref](./images/editing_app_setting/10_configuration_ref.png)
 
 **NOTE:** After the setting **my_tasks_filters** is a setting **launch_at_startup**. This is important to note because it’s the default setting for the **File Open** setting in the project environment. This tells you that when Maya is launched from the Shotgun Desktop, the **File Open** dialog box automatically opens.
```
Comment (pscadding) [inline docs/en/toolkit/learning-resources/guides/editing_app_setting.md]: I know this isn't a change, but should this be a note box?

Comment (pscadding) [reply docs/en/toolkit/learning-resources/guides/editing_app_setting.md]: Also again not a change you've made but I spotted the last sentence in the doc doesn't have a full stop.

> Now that you’ve learned how to modify an app configuration setting, try adding an app to your Toolkit configuration

Comment (tannazie) [reply docs/en/toolkit/learning-resources/guides/editing_app_setting.md]: changed that note and another one to a note box; good catch on the missing period! fixed.

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/e59af8a62326d29b53f61815e560b47e261aadf0/index.html)

**Pull request merged**